float64 dt
float64 v_des
float64[] q_des
int64 rate
bool verbose
---
bool success
